kafka-native/src/main/java/org/ballerinalang/messaging/kafka/impl/KafkaListenerImpl.java

```diff
@@ -46,13 +45,13 @@
  */
 public class KafkaListenerImpl implements KafkaListener {
 
-    private Scheduler scheduler;
     private BObject service;
     private BObject listener;
     private ResponseCallback callback;
+    private BRuntime bRuntime;
 
-    public KafkaListenerImpl(Strand strand, BObject listener, BObject service) {
-        this.scheduler = strand.scheduler;
+    public KafkaListenerImpl(Strand strand, BObject listener, BObject service, BRuntime bRuntime) {
```

Comment:
Can we remove the `strand` now?

Reply:
Yes. Fixed in f5efda80b5c70e350253b1fe80f70f7a79f9b9f7